src/components/Votes/VoteDisplay/VoteDisplay.scss

```diff
@@ -1,29 +1,139 @@
 @import "src/constants/style.scss";
 
+$vote-display-height: 42px;
+$vote-display-width: 114px;
+
 .vote-display {
+  height: $vote-display-height;
+  width: $vote-display-width;
+  color: $color-white;
+  background-color: $menu-icon-background-color--dark;
+  border-radius: calc($vote-display-height / 2);
   position: relative;
-  width: 85px;
+  border: 2px solid $menu-icon-background-color--dark;
 }
 
-.vote-display > * {
-  color: $color-white;
-  background-color: $color-backlog-blue;
-  vertical-align: bottom;
-  box-shadow: 0 8px 16px #0057ff25;
+.vote-display > svg {
+  position: absolute;
+  height: 100%;
+  background-color: $menu-icon-background-color--dark;
+  border-radius: 100%;
 }
-
-[theme="dark"] {
-  .vote-display > * {
-    background-color: $color-planning-pink;
-    box-shadow: 0 8px 16px #e2036025;
-  }
+[theme="dark"] .vote-display > svg {
+  background-color: $color-white;
+  color: $color-black;
 }
+
 .vote-display > span {
-  display: inline-block;
+  position: absolute;
+  right: 0;
+  left: 0;
+  height: 100%;
+  width: 100%;
+  padding-left: 38px;
+  line-height: 38px;
+  text-align: center;
+  font-weight: bold;
+}
+
+.vote-display__progress-bar {
+  position: absolute;
+  top: 0;
+  left: 19px;
+  height: 38px;
+  border-top-right-radius: 19px;
+  border-bottom-right-radius: 19px;
+  background-color: $color-progress-circle;
+}
+
+.vote-display__short-actions {
+  height: 100%;
+  width: 100%;
+  position: absolute;
+  padding-left: 38px;
+  border-radius: 19px;
+  background-color: $menu-icon-background-color--dark;
+  visibility: hidden;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: $margin--small;
+}
+
+.vote-display:hover > .vote-display__short-actions {
+  visibility: visible;
+}
+
+.short-actions__button-wrapper {
+  position: relative;
+  height: 28px;
+  width: 28px;
+  border-radius: 14px;
+}
+
+.short-actions__button-wrapper > button {
+  height: 28px;
+  width: 28px;
+  border-radius: 14px;
+  background-color: $color-dark-gray;
+  color: $menu-icon-background-color--dark;
+  border: none;
+  padding: 0;
+  cursor: pointer;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.short-actions__button-wrapper > button > svg {
+  height: $icon--small;
+  width: $icon--small;
+}
+.short-actions__button-wrapper > button:hover {
+  background-color: $color-progress-circle;
+  color: $color-white;
+}
+
+// Short Actions Button Tooltip
+.short-actions__button-wrapper > span {
+  visibility: hidden;
+  opacity: 0;
+  transition: opacity 150ms linear;
+  position: absolute;
+  left: 50%;
+  transform: translateX(-50%);
+  height: 36px;
   border-radius: 18px;
-  width: 84px;
-  cursor: default;
+  padding: 5px $padding--default;
```

Comment:
Is this so that the tooltip is not overlapped by the vote-display icon?

Reply:
No, there's no other reason behind this padding except for the padding of the tooltip container itself